CPP17.md

```diff
@@ -150,6 +150,17 @@ S x2 = S{123};        // mov eax, dword ptr [.L_ZZ4mainE2x2]
                       // .L_ZZ4mainE2x2: .long 123
 ```
 
+It can also be used to declare and define a member variable, such that it does not need to be initialized in the source file.
```

Comment:
I think the following is more accurate: `declare and define a static member variable`. I added "static" in there to make it clear this feature applies to static members.